README.md

```diff
@@ -122,6 +129,20 @@ Newer versions of Android–`API Level 23` (Android 6.0)–have a different perm
 
 To see this library in action, check out the [opentok-react-native-samples](https://github.com/opentok/opentok-react-native-samples) repo.
 
-## Contributing
+## Development and Contributing
+
+Interested in contributing? We :heart: pull requests! See the
+[Contribution](CONTRIBUTING.md) guidelines.
+
+## Getting Help
+
+We love to hear from you so if you have questions, comments or find a bug in the project, let us know! You can either:
+
+- Open an issue on this repository
+- See <https://support.tokbox.com/> for support options
+- Tweet at us! We're [@VonageDev](https://twitter.com/VonageDev) on Twitter
+- Or [join the Vonage Developer Community Slack](https://developer.nexmo.com/community/slack)
+
+## Further Reading
 
-If you make changes to the project that you would like to contribute back then please follow the [contributing guidelines](CONTRIBUTING.md). All contributions are greatly appreciated!
+- Check out the Developer Documentation at <https://tokbox.com/developer/>
```

Comment:
The documentation for this library is within this repo, it's not on developer center. We should remove this line or improve it and say it's in the repo. WDYT?

Reply:
Agreed.  Removed this section.